src/chain_spec.rs

```diff
@@ -112,10 +145,17 @@ where
 }
 
 pub fn centrifuge_config() -> CentrifugeChainSpec {
-	CentrifugeChainSpec::from_json_bytes(
+	let mut spec = CentrifugeChainSpec::from_json_bytes(
 		&include_bytes!("../res/genesis/centrifuge-genesis-spec-raw.json")[..],
 	)
-	.unwrap()
+	.unwrap();
+	let extension = spec.extensions_mut();
+	*extension = Extensions {
+		relay_chain: "polkadot".into(),
+		para_id: 2031,
```

Comment:
We can use `parachains::polkadot::centrifuge::PARA_ID` here and same for similar const usages 👍